spec/controllers/animals_ controller_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'spec_helper'
+
+describe Api::V1::AnimalsController do
+  let!(:user) { FactoryGirl.create(:user, account_active: 'true', permissions: 'animals_edit') }
+  let!(:animal) { FactoryGirl.create(:animal) }
+   
+  before(:each) do
+    @request.env['devise.mapping'] = Devise.mappings[:user]
+  end
+
+ 	describe 'POST #create' do
+  
+    let(:name) { animal.name }
+    let(:chip_num) { Faker::Code.isbn }
+    let(:race) { animal.race }
+    let(:sex) { animal.sex }
+    let(:vaccines) { animal.vaccines }
+    let(:castrated) { animal.castrated }
+    let(:admission_date) { animal.admission_date }
+    let(:birthdate) { animal.birthdate }
+    let(:species_id) { animal.species_id }
+
+    let(:params) do  
+      { 
+        name: name, 
+        chip_num: chip_num, 
+        race: race, 
+        sex: sex, 
+        vaccines: vaccines, 
+        castrated: castrated, 
+        admission_date: admission_date, 
+        birthdate: birthdate, 
+        species_id: species_id 
+      } 
+    end
+      
+    context 'de un animal' do
+      context 'exitoso' do
+
+        it 'devuelve código' do
+          sign_in user
```

Comment:
el `sign_in user` y la linea de abajo lo repetis en todos los `it`, porque no lo pones en un `before_each` en vez de repetir codigo?